React InstantSearch/injected-content/src/InjectedInfiniteHits.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { createClassNames, connectInfiniteHits } from 'react-instantsearch-dom';
+
+import { connectInjectedHits } from './connectInjectedHits';
+
+const cx = createClassNames('Hits');
+
+export const InjectedInfiniteHits = connectInfiniteHits(
```

Comment:
or is it InfiniteInjectedHits?

Reply:
Could be both, I don't have a preference. We've called it `InjectedInfiniteHits` in the guide, so if we want to change, let's do it in both places after we merge.